README.rst

```diff
@@ -41,8 +41,30 @@ Control machine requirements:
 
 - Ansible 2.1+ installed. See `Ansible install docs`_
 
+Initial set-up
+--------------
+First, clone the jupyterhub-deploy-teaching repository to your machine and navigate to the folder.
+
+.. code-block:: 
+
+   git clone https://github.com/LibreTexts/jupyterhub-deploy-teaching
+   cd jupyterhub-deploy-teaching
+   
+There are two options for configuring and deploying JuptyerHub.
+
+Option 1: Automatic configuration and deployment
+------------------------------------------------
+After cloning the repository, you may run `bash test_setup.sh` to configure and deploy Jupyterhub. 
```

Comment:
`test_setup.sh` doesn't seem like an obvious script name. Maybe `configure_and_deploy.sh`? or `setup.sh`.

Reply:
Changed the file name to setup.sh